frontend/src/metabase/ui/components/inputs/SegmentedControl/SegmentedControl.styled.tsx

```diff
@@ -0,0 +1,54 @@
+import type {
+  MantineThemeOverride,
+  SegmentedControlStylesParams,
+} from "@mantine/core";
+import { getSize, rem } from "@mantine/core";
+
+const OPTION_PADDING = {
+  xs: rem(8),
+  sm: rem(8),
+  md: rem(8),
+};
+
+const OPTION_PADDING_FULL_WIDTH_V = {
+  xs: rem(8),
+  sm: rem(8),
+  md: rem(8),
+};
+
+const OPTION_PADDING_FULL_WIDTH_H = {
+  xs: rem(16),
+  sm: rem(16),
+  md: rem(16),
+};
+
+export const getSegmentedControlOverrides =
+  (): MantineThemeOverride["components"] => ({
+    SegmentedControl: {
+      defaultProps: {
+        size: "md",
+        radius: rem(4),
+      },
+      styles: (
+        theme,
+        { fullWidth }: SegmentedControlStylesParams,
+        { size = "md" },
+      ) => {
+        return {
+          label: {
+            padding: fullWidth
+              ? `${getSize({
```

Comment:
I see no point in using [`getSize`](https://github.com/mantinedev/mantine/blob/v6/src/mantine-styles/src/theme/utils/get-size/get-size.ts), if all sizes have the same dimension (see `OPTION_PADDING`, `OPTION_PADDING_FULL_WIDTH_V`, `OPTION_PADDING_FULL_WIDTH_H`).

Would it make more sense to just inline `rem(8)` or `rem(16)` instead?